scientific/src/commonTest/kotlin/unit/DimensionlessUnitTest.kt

```diff
@@ -0,0 +1,131 @@
+/*
+ Copyright 2022 Splendo Consulting B.V. The Netherlands
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+ */
+
+package com.splendo.kaluga.scientific.unit
+
+import com.splendo.kaluga.base.utils.toDecimal
+import com.splendo.kaluga.scientific.converter.decimal.div
+import com.splendo.kaluga.scientific.converter.decimal.times
+import com.splendo.kaluga.scientific.converter.dimensionless.div
+import com.splendo.kaluga.scientific.converter.dimensionless.times
+import com.splendo.kaluga.scientific.invoke
+import kotlin.test.Test
+import kotlin.test.assertEquals
+
+class DimensionlessUnitTest {
+    @Test
+    fun oneUnitConstantTest() {
+        val one = One.constant
+        val result = one.decimalValue
+        val expected = 1.0.toDecimal()
+        assertEquals(expected, result)
+    }
+
+    fun oneUnitFromDecimalTest() {
+        val base = 123.0(One)
```

Comment:
123 one...

Reply:
Nitpick: 
Measurement = magnitude * unit +/- uncertainty
so this says: "I've measured 123.0 occurrences of unit One"
and is absolutely valid statement

Reply:
Okay, clear